src/components/MenuItem/styled/li.js

```diff
@@ -17,24 +26,24 @@ const StyledLi = styled.li`
     justify-content: space-between;
     align-items: center;
     padding: 0.75rem 0.75rem;
-    color: ${COLOR_DARK_1};
+    color: ${props => props.mainText};
     white-space: nowrap;
     cursor: pointer;
     text-decoration: none;
     box-sizing: border-box;
 
     &:focus {
         outline: 0;
-        background-color: ${COLOR_GRAY_1};
+        background-color: ${props => props.hover};
     }
 
     &:active {
-        background-color: ${COLOR_GRAY_1};
+        background-color: ${props => props.hover};
```

Comment:
Maybe we can use other action for this like active and use it here and above when focus since hover is not semantic to focus and active